include/RAJA/policy/cuda/forallN.hpp

```diff
@@ -68,18 +68,6 @@ struct ForallN_BindFirstArg_Device {
 };
 
 
-RAJA_INLINE
```

Comment:
Is this needed for some forallN use cases?

Never mind. I think you just moved it to someplace else.